lib/engine.js

```diff
@@ -18,6 +18,16 @@ const Mailer = require('inxt-service-mailer');
 const log = require('./logger');
 const ComplexClient = require('storj-complex').createClient;
 const MongoDBStorageAdapter = require('storj-mongodb-adapter');
+const NetworkMessageQueue = require('./server/queues/networkQueue');
+
+const {
+  QUEUE_HOST: HOST,
+  QUEUE_PORT: PORT,
+  QUEUE_USERNAME: USER,
+  QUEUE_PASSWORD: PASS,
+} = process.env;
```

Comment:
Unsure if it's ok to import them right here or to create a file to import them

Reply:
Right now we will keep the same style the project has, although is not the best, until we change it entirely